src/pages/product-details/product-images/product-images.styles.js

```diff
@@ -103,7 +104,7 @@ export const useStyles = makeStyles((theme) => ({
   lastImage: {
     ...imgSides,
     filter: 'brightness(50%)',
-    boxShadow: theme.palette.carouselItem.shadow.boxShadow
+    boxShadow: 'inset 0 0 0 1000px rgba(0,0,0,.5)'
```

Comment:
Isn't it better idea to create another instance of shadow in palette and move the value there?

Reply:
fixed